environment/config.go

```diff
@@ -39,6 +39,10 @@ import (
 	"github.com/m3db/m3x/instrument"
 )
 
+const (
+	defaultSDTimeout = 5 * time.Second
```

Comment:
Perhaps 10s is a better timeout, also this might cause problems for newly created clusters that don't have a namespace setup, especially if we need to call endpoints on the nodes themselves to create the first namespace (humans don't want to race this timestamp to create a cluster).

Reply:
hmm seems this timeout is applied to the service watch not the namespace one? Also do we plan to manually create the initial placement? could that info somehow be passed in through config and the process creates the placement itself?